weave/guides/platform/weave-self-managed.mdx

```diff
@@ -13,86 +13,64 @@ Self-managed Weave deployments rely on [ClickHouseDB](https://clickhouse.com/) t
 
-- **Security & Compliance**: Adjust security contexts, runAsUser/fsGroup values, and other security settings according to your organization's security policies and Kubernetes/OpenShift requirements.
+- **Security & Compliance**: Security contexts, `runAsUser`/`fsGroup` values, and other security settings according to your organization's security policies and Kubernetes/OpenShift requirements.
 - **Resource Sizing**: The resource allocations shown are starting points. **Consult with your W&B Solutions Architect team** for proper sizing based on your expected trace volume and performance requirements.
 - **Infrastructure Specifics**: Update storage classes, node selectors, and other infrastructure-specific settings to match your environment.
 
-Each organization's Kubernetes environment is unique - treat these configurations as templates to be adapted, not prescriptive solutions.
-</Warning>
+These configurations in this guide should be treated like templates, not prescriptive solutions.
 
 ## Architecture
 
-```
-┌─────────────────────────────────────────────────────────────┐
-│                    W&B Platform (wandb)                     │
-│  ┌────────────┐  ┌────────────┐  ┌────────────────────┐     │
-│  │ weave-trace│──│  app/api   │──│  console/parquet   │     │
-│  └──────┬─────┘  └────────────┘  └────────────────────┘     │
-└─────────┼────────────────────────────────────────────────---┘
-          │
-          ▼
-┌─────────────────────────────────────────────────────────────┐
-│              ClickHouse Cluster (clickhouse)                │
-│  ┌──────────────┐  ┌──────────────┐  ┌──────────────┐       │
-│  │ ch-server-0  │  │ ch-server-1  │  │ ch-server-2  │       │
-│  └──────┬───────┘  └──────┬───────┘  └──────┬───────┘       │
-│         │                 │                 │               │
-│         └─────────────────┼─────────────────┘               │
-│                           │                                 │
-│  ┌──────────────────────────────────────────────────────┐   │
-│  │           ClickHouse Keeper Cluster                  │   │
-│  │  ┌──────────┐  ┌──────────┐  ┌──────────┐            │   │
-│  │  │ keeper-0 │  │ keeper-1 │  │ keeper-2 │            │   │
-│  │  └──────────┘  └──────────┘  └──────────┘            │   │
-│  └──────────────────────────────────────────────────────┘   │
-└─────────┬───────────────────────────────────────────────────┘
-              │
-              ▼
-      ┌───────────────┐
-      │  S3 Storage   │
-      │  (AWS/MinIO)  │
-      └───────────────┘
+```mermaid
+graph TD
+    A["W&B Platform (wandb)<br/>weave-trace · app/API · console/parquet"] --> B["ClickHouse Cluster"]
+    B --> C["ch-server-0"]
+    B --> D["ch-server-1"]
+    B --> E["ch-server-2"]
+    C --> F["ClickHouse Keeper Cluster<br/>keeper-0 · keeper-1 · keeper-2"]
+    D --> F
+    E --> F
+    F --> G["S3 Storage<br/>(AWS/MinIO)"]
```

Comment:
The bucket needs to be connected to the ClickHouse Cluster, not Keeper. My previous architecture wasn’t completely correct, I tried to generalize the entire cluster connected to the Bucket